connectors-element-template-links.txt

```diff
@@ -120,6 +124,7 @@ https://docs.camunda.io/docs/components/connectors/use-connectors/
 https://docs.camunda.io/docs/components/connectors/use-connectors/#bpmn-errors
 https://docs.camunda.io/docs/components/connectors/use-connectors/#result-expression
 https://docs.camunda.io/docs/components/connectors/use-connectors/#result-variable
+https://docs.camunda.io/docs/components/console/manage-clusters/setup-client-connection-credentials
```

Comment:
This has not moved in the current version of docs, but has moved to the location above in the `next` version of docs.

Reply:
I think keeping it as `/docs/guides/setup-client-connection-credentials/` is correct as this is the current version of docs 👍